kotlin-native/build-tools/build.gradle.kts

```diff
@@ -42,11 +42,8 @@ repositories {
     maven(kotlinCompilerRepo)
     maven(buildKotlinCompilerRepo)
     maven("https://cache-redirector.jetbrains.com/maven-central")
-    mavenCentral()
-    maven("https://kotlin.bintray.com/kotlinx")
-    maven("https://dl.bintray.com/kotlin/kotlin-dev")
+    maven("https://cache-redirector.jetbrains.com/dl.kotlin.bintray.com/kotlinx")
     maven("https://cache-redirector.jetbrains.com/jcenter")
-    jcenter()
```

Comment:
(occasionally deleted comment) Cleanup cache-redirector, also we add jcenter to all projects in the root buildscript